internal/repohttp/handler_test.go

```
@@ -0,0 +1,128 @@
+		router:   router,
+		store:    store,
+		repo:     repo,
+		repoPath: repoPath,
+		apiPath:  "/api/v1/orgs/" + org.Slug + "/projects/" + project.Slug + "/repos/" + repo.Slug,
+		token:    token,
+	}
+}
+
+func (f deleteFixture) delete(t *testing.T) int {
+	t.Helper()
+	req := httptest.NewRequest(http.MethodDelete, f.apiPath, nil)
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)